ontogen/base/__init__.py

```
@@ -0,0 +1,171 @@
+from abc import ABCMeta, abstractmethod
+from typing import List, Type
+
+from owlready2 import Thing, ClassConstruct
+from owlready2.prop import destroy_entity
+
+from .assertable import OwlAssertable
+from .ontology import Ontology
+from .vars import BUILTIN_NAMES, DATATYPE_MAP, GENERATED_TYPES, LABEL_ENTITY_NAME, COMMENT_ENTITY_NAME, BUILTIN_DATA_TYPES
+
+
+def cleanup(onto: Ontology):
+    onto.implementation.graph.destroy()
+    for e in GENERATED_TYPES:
+        try:
+            destroy_entity(GENERATED_TYPES[e])
+        except:
+            pass
+    GENERATED_TYPES.clear()
```

Why is there a Pokemon exception here? Are you sure this won't hide any errors?